casper/src/test/scala/coop/rchain/casper/MultiParentCasperRholangSpec.scala

```diff
@@ -53,8 +53,7 @@ class MultiParentCasperRholangSpec extends FlatSpec with Matchers with Inspector
   }
 
   it should "be able to use the registry" in effectTest {
-    HashSetCasperTestNode.standaloneEff(genesis, validatorKeys.head).use { implicit node =>
-      import node.casperEff
+    HashSetCasperTestNode.standaloneEff(genesis, validatorKeys.head).use { node =>
       implicit val rm = node.runtimeManager
 
       val registerDeploy = ConstructDeploy.sourceDeploy(
```

Comment:
A simple alternative to fixing a timestamp is to just use `RegistrySigGen` to calculate the URI.

Reply:
Will address that in an even more elegant manner after we have `deployId` merged